pom.xml

```diff
@@ -68,6 +68,12 @@
             <artifactId>httpclient</artifactId>
             <version>4.5.7</version>
         </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
```

Comment:
The testng dependency already exists.

Reply:
I've deleted testng dependency
<!-- https://mvnrepository.com/artifact/org.testng/testng -->
        <!--<dependency>-->
            <!--<groupId>org.testng</groupId>-->
            <!--<artifactId>testng</artifactId>-->
            <!--<version>6.14.3</version>-->
            <!--<scope>test</scope>-->
        <!--</dependency>-->